### DISCUSSION TITLE
2024年开发组休假通知

### DISCUSSION THREAD

**Discussion by PythonSmall-Q** [XMOJ-Script-dev/XMOJ-Script#334] [Announcements]
2024年，开发组和管理组预定休假如下：

2024年1月6日0：00-2024年1月11日23：59休假

新年期间（按照国务院发布的时间）休假

注：由于后续开发和维护周期未知，2024年后续休假安排待定

休假期间暂停功能开发维护，所有的bug和issue等到休假结束后统一处理，但是不会撤销权限。开发组、管理组成员额外休假将会撤销权限。

**Comment by langningchen** [comment]
@PythonSmall-Q LOL
